apcd-cms/src/client/src/components/Submitter/Exceptions/ExceptionForm.tsx

```diff
@@ -0,0 +1,339 @@
+import React, { useState, useEffect } from 'react';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import * as Yup from 'yup';
+import {
+  FormGroup,
+  Label,
+  Input,
+  Button,
+  InputGroup,
+  FormFeedback,
+} from 'reactstrap';
+import { cdlObject, useCDLs, cdl } from 'hooks/cdls';
+import { Entities, useEntities } from 'hooks/entities';
+import styles from './ExceptionForm.module.css';
+import LoadingSpinner from 'core-components/LoadingSpinner';
+import SectionMessage from 'core-components/SectionMessage';
+
+//Boiler plate example for validation schema
+
+const validationSchema = Yup.object({
+  name: Yup.string()
+    .max(64, 'Must be 64 characters or less')
+    .required('Required'),
+  id: Yup.string().required('Required'),
+  year: Yup.number().required('Required'),
+});
+
+interface FormValues {
+  name: string;
+  id: string;
+  year: number;
+}
+
+const initialValues: FormValues = {
+  name: '',
+  id: '',
+  year: 2024,
+};
```

Comment:
For future PR, these should be something like:

```suggestion
interface FormValues {
  businessName_1: string;
  file_type_1: string;
  fieldThresholdException_1: string;
  expiration_date_1: string;
  requested_threshold_1: number | '';
  required_threshold_1: number | '';
}

const initialValues: FormValues = {
  businessName_1: '',
  file_type_1: '',
  fieldThresholdException_1: '',
  expiration_date_1: '',
  requested_threshold_1: '',
  required_threshold_1: '',
};
```